detail.js

```diff
@@ -74,13 +74,12 @@ var app = new Vue({
 
                 })
                 .catch(error => console.error("Unable to get manta rays.", error));
-            
-                
+
 
         },
         GetImagePath: function (mantaray)
         {
-            fetch(this.imageURI + mantaray._mantaID)
+            fetch(this.imageURI + mantaray._mantaID, {cache: "no-store"})
```

Comment:
Were you having caching issues here? Unless the image path changes a lot, may make more sense to let the browser cache this as much as possible. One less round trip to your API means faster load times.
This is true in all locations you've added the no-store cache header.

Reply:
I am still caching the Photo's themselves, I am just not caching the list of Photo Objects for a manta. Since a new photo of a manta can get added and users will want to see it sooner rather than later, we don't want to cache the list of available Photos of that manta. The Photo's themselves (and their Path's) won't change much at all so caching them is desired.
Same applies to the list of Sightings. If a new one get's added, we'll want to see it the next time anyone looks. We'll want to always be in sync with what is in the Database so they shouldn't be cached.
These objects are small and don't, relatively, take much time to download.